### PR TITLE
README: add deployment information

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

This repo had no documentation for how to deploy changes to AWS;  these changes to the README fill in that gap.

## Was the documentation (README, API, wiki, ...) updated?

yes, README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +148,60 @@ $ rake mappings:sort
...
+- "Save Draft"  (NOT:  "Publish Release")
+
+2. On your laptop, create a tag and push it to github
+
+```
+git tag 1.1.3
+git push origin --tags
+```
+
+3.  Go to https://github.com/sul-dlss/dlme/releases/new?tag=1.1.3 and "Publish Release".
```
Comment (mjgiarlo) [inline README.md]: Should `dlme` here be `dlme-transform`?

Comment (ndushay) [reply README.md]: yes!  thanks

Comment (ndushay) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -143,6 +148,60 @@ $ rake mappings:sort
...
+
+#### Deploy New Tagged Release to AWS
+
+Terraform tells AWS which tagged image of dlme-transform to use for staging and production environments.
+
+In the case of this code base, dlme-transform, a PR to https://github.com/sul-dlss/terraform-aws must be made that updates the "image" variable for dlme-transform, e.g. `"suldlss/dlme-transform:1.1.3"`.  For an example, see https://github.com/sul-dlss/terraform-aws/pull/525.
+
+Some more info here https://github.com/sul-dlss/terraform-aws/blob/master/organizations/development/dlme/README.md
```
Comment (mjgiarlo) [inline README.md]: Should this be linking to the staging and/or production READMEs since this is the staging/prod section of the README?

Comment (ndushay) [reply README.md]: Also a good catch, and also fixed.

**Pull request merged**